dockerfiles/backfill_storage/startup_script.sh

```diff
@@ -28,5 +28,5 @@ if test -z "$VDB_PG_CONNECT"; then
 fi
 
 # Run getStorageValue
-echo "Running backfillStorage from block <NEEDS UPDATE> to <NEEDS UPDATE>"
-./vulcanizedb backfillStorage -s=10739859 -e=10906308 --config config.toml # TODO: update block numbers before next run
+echo "Running backfillStorage from block 109326661 to 10970892"
+./vulcanizedb backfillStorage -s=109326661 -e=109708921 --config config.toml # TODO: update block numbers before next run
```

Comment:
nbd but it seems like the ending block numbers differ by 1 between the logs and the config

Reply:
also I know this sorta breaks precedent but I'd be in favor of starting 1 block before the deploy - can't really explain why but it has something to do with guaranteeing that we get all the deployment stuff, and I think it should be super low cost (process is lightweight on blocks without diffs)